Sources/SymDiff/source/Config.cs

```diff
@@ -43,10 +43,13 @@ public class ProcedureMap : List<Duple<HDuple<string>, ParamMap>>
   {
     public readonly string prefix;
 
+    public Dictionary <string, string> equivalentProcs;
+
```

Comment:
This seems to only be read and never updated. Is the idea that a client would update it directly?

Reply:
Oh, indeed, good point. Its better to add a store function, I agree.